src/test/scala/edu/ie3/simona/agent/em/EmAgentIT.scala

```diff
@@ -150,7 +150,7 @@ class EmAgentIT
               pvInput,
               PvRuntimeConfig(
                 calculateMissingReactivePowerWithModel = true,
-                scaling = 2d,
+                scaling = 1d,
```

Comment:
I missed this when I changed it. Since default parameter for scaling is 1, this line could be removed. 

Same for scaling and initicalSoc for Storage below. Could you please remove them as well? Thanks!